packages/apps/docs/src/components/Layout/Blog/styles.ts

```diff
@@ -1,6 +1,6 @@
 import { styled, StyledComponent } from '@kadena/react-components';
 
-import { BasePageGrid } from '../components';
```

Comment:
layout looks weird on small screen.
different as in browser

![image](https://github.com/kadena-community/kadena.js/assets/4015521/38a316c5-afbc-416c-8143-b9124c6e168e)

Reply:
As we discussed IRL, https://app.asana.com/0/1205288273408141/1205384296981150/f created a separate bug ticket for fixing this